tests/integration/test_distributeby.py

```diff
@@ -0,0 +1,30 @@
+import dask.dataframe as dd
+import pandas as pd
+
+
+def test_distribute_by(c):
```

Comment:
We might want to parametrize this to also run a GPU version of the test, which I think can be done by switching out the library used to create `df_0`/`df_1`.